projects/js-packages/publicize-components/src/components/social-image-generator/panel/index.js

```diff
@@ -69,7 +67,7 @@ const SocialImageGeneratorPanel = ( { prePublish = false } ) => {
 	};
 
 	return (
-		<PanelWrapper { ...wrapperProps }>
+		<PanelBody title={ __( 'Social Image Generator', 'jetpack' ) }>
```

Comment:
This change makes the layout of the pre-publish side bar a bit odd. There's a gap at the bottom of the panel, but also this panel becomes nested in the 'Share this post' one.

![CleanShot 2023-02-23 at 10 38 00@2x](https://user-images.githubusercontent.com/96462/220884132-333f397f-70a2-4d64-b55d-25006d467ed5.png)

<img width="307" alt="image" src="https://user-images.githubusercontent.com/96462/220884295-08d056f2-e805-4981-8b8e-c8e219ed0b0c.png">

Reply:
Yeah good catch. Following our conversation on Slack, I restored the previous version, and added it to the Jetpack side as well. It is duplicated like this for now, but it will be removed with the next UI change anyway

Reply:
Yes, I think that makes sense for now. Thanks!